tests/test_interface_switchport.rb

```diff
@@ -675,3 +386,268 @@ def test_interface_svi_command_on_non_vlan
     assert_raises(RuntimeError) { interface.svi_management = true }
   end
 end
+
+# TestInterfaceSwitchportSvi
+# Minitest for Interface switchport configuration in combo with interface-vlan
+# Not applicable to IOS XR
+class TestInterfaceSwitchportSvi < TestInterfaceSwitchport
+  attr_reader :svi
+
+  def setup
+    super
+    skip('VLAN interfaces are not supported on IOS XR') if platform == :ios_xr
+    @svi = Interface.new('Vlan23')
+  end
+
+  def teardown
+    svi.destroy unless platform == :ios_xr
```

Comment:
No can do - if we're on XR, svi doesn't ever get created (it's nil), so simply trying to call destroy raises an error.